<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>audio_player</class>
 <widget class="QWidget" name="audio_player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>56</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#frame_player {
	background-color: #151515;
	border-radius: 6px;
}

QPushButton {
	background: none;
	border: none;
	color: #808080;
}
QPushButton[highlight=&quot;true&quot;] {color: white;}

#btn_file {
	padding-left: 5px;
	text-align: middle left;
	font-size: 11pt;
	font-weight: 600;
	color: white;
}
#btn_file:disabled {color: #A0A0A0}
#btn_file:enabled:hover {text-decoration: underline;}

#slider_volume {max-width: 100px;}

QSlider {padding: 4px;}
QSlider::groove {
	background-color: #808080;
	border-radius: 2px;
}
QSlider::add-page:vertical {
	background-color: white;
	border-bottom-left-radius: 2px;
	border-bottom-right-radius: 2px;
	margin-top: -1px;
}
QSlider::sub-page:horizontal {
	background-color: white;
	border-top-left-radius: 2px;
	border-bottom-left-radius: 2px;
}
QSlider::handle {
	background-color: white;
	width: 4px;
	height: 4px;
	border-radius: 2px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="layout_widget">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_player">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>56</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>56</height>
      </size>
     </property>
     <layout class="QGridLayout" name="layout_player">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="1">
       <widget class="QSlider" name="slider_playback">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>12</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QFrame" name="frame_volume">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="layout_volume">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSlider" name="slider_volume">
           <property name="maximumSize">
            <size>
             <width>108</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="IconButton" name="btn_volume">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::AudioVolumeHigh"/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="IconButton" name="btn_play">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QPushButton" name="btn_file">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">Filename</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IconButton</class>
   <extends>QPushButton</extends>
   <header>widgets.common</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
